packages/private-apis/src/implementation.js

```diff
@@ -27,6 +27,7 @@ const CORE_MODULES_USING_PRIVATE_APIS = [
 	'@wordpress/format-library',
 	'@wordpress/patterns',
 	'@wordpress/reusable-blocks',
+	'@wordpress/rich-text',
```

Comment:
@ellatrix I'm opting RichText into privateAPIs. Just FYI

Reply:
I've reverted this as I discovered a new way which is already in use in the text color format:

https://github.com/WordPress/gutenberg/pull/57726/commits/160f4d3c46bae1f24b89dc4473e754b36c5c6622